data/tools/nebula.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "slug": "Nebula",
+    "name": "Nebula",
+    "description": "Nebula is a Docker orchestration tool designed to manage massive scale distributed clusters",
+    "url": "http://nebula.readthedocs.io/en/latest/",
+    "tags": [
+      "linux",
+      "open-source",
+      "virt",
+      "orchestration",
+      "pyhton"
```

Comment:
Created #170 with a fix, honestly don't know why I keep making this same typo over and over...